README.md

```diff
@@ -448,56 +449,50 @@ Potentially, these could be used later to help define the COALA IP specification
-others to subclass their *core* schemata into what it calls "*hosted*" and "*external*" extensions.
-There are three basic types of schemata on schema.org:
+Despite an exhaustive list of schemata provided by schema.org, we still have a few use cases that
+have not been covered: the missing `Right` and `RightsConflict` entity types, as well as any
+additional RRM properties that have not been defined in the existing schemata. Thankfully,
+schema.org was designed with extensibility in mind–we can modify existing, and even create new,
+schemata to fit our needs. Schema.org even encourages others to subclass the *core* schemata into
+"*hosted*" and "*external*" extensions, making available three types of schemata:
 
-- **Core:** A basic vocabulary for describing the kind of entities most common web applications
-  need.
-- **Hosted:** Subclassed models from Core which have their own namespace on schema.org (e.g.
-  http://health-lifesci.schema.org/) and are reviewed by the schema.org community. Hosted schemata
-  should be application-agnostic.
-- **External:** Subclassed models from Core or Hosted which have an application-specific namespace (e.g.
-  http://schema.coala.global). External schemata may be application-specific.
+- **Core:** A set of basic vocabulary for describing the kind of entities needed by most web
+  applications;
+- **Hosted:** Application-agnostic schemata deriving from *core* which have their own namespace on
+  schema.org (e.g. http://health-lifesci.schema.org/) and are reviewed by the schema.org community;
+  and
+- **External:** Schemata from *core* or *hosted* which have an application-specific namespace (e.g.
+  http://schema.coala.global) and may be application-specific
 
 
-Applied to the context of the COALA IP specification, application-agnostic schemata (including all
-of LCC RRM) would ideally become a *hosted* extension. Application-specific schemata, data models
-that are specific for a specific application or service, would become *external* schemata.
-Fortunately, leveraging schema.org in this way maintains compliance with rules
-five and six of the LCC's "Ten Targets":
+In the context of COALA IP, any application-agnostic schemata, including all [schemata derived from
+the LCC RRM](#coala-ip-implementing-the-lcc-rrm-with-linked-data), would become a *hosted*
+extension. Fortunately, leveraging schema.org in this way maintains compliance with rules five and
+six of the [LCC's "Ten Targets"](#the-lccs-ten-targets):
 
-- Rule 5: Links between identifiers are system agnostic and need to be authorized by participating
+- **Rule 5**: Links between identifiers are system agnostic and must be authorized by participating
   consortiums.
-- Rule 6: Metadata is system agnostic and its schema has to be authorized by participating parties
+- **Rule 6**: Metadata is system agnostic and its schema must be authorized by participating parties
```

Comment:
Nice catch; I've opted to copy the framing from above. See f8dbfb5934d53967fab2d3c41e1e021b576e9436.